docs/foundations/omsf.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: Open Molecular Software Foundation - How to Join
+---
+
+*February 2025*
+
+*Zachary Baker*  
+
+  
+### **About OMSF**
+
+The Open Molecular Software Foundation (OMSF) is a nonprofit organization that provides governance, infrastructure, and financial support for open-source projects in molecular sciences. We help researchers, developers, and industry partners build sustainable, high-impact software by offering operational resources, community engagement, and long-term strategic guidance.
+
+We work with academics looking for new ways to make their work sustainable, industries that want to promote open science, and community-based projects focused on events, education, and stewardship.
```

Comment:
maybe rather than "academics" you could say "researchers"? To include researchers who are not in academic institutions, such national labs...